blog-server-services/src/impls/rbatis_author_service.rs

```diff
@@ -20,15 +20,15 @@ impl_select!(Author {select_by_telegram_id(telegram_id: &u64) -> Option =>
 impl_select!(Author {select_by_slug(slug: &String) -> Option => 
     "`WHERE slug = #{slug} LIMIT 1`"});
 impl_select!(Author {select_all_with_offset_and_limit(offset: &u64, limit: &u64) => 
-    "`LIMIT #{limit} OFFSET #{offset}`"});
+    "`ORDER BY author.editor DESC, author.blocked ASC, author.registered_at ASC LIMIT #{limit} OFFSET #{offset}`"});
```

Comment:
do we even need to display blocked users?
could we just filter this records out?

Reply:
yep, let's keep at the end